### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by ildede**
As mention [here](https://github.com/shpakovski/MASShortcut/issues/145#issuecomment-633193566).
This PR is a review of the Italian translation to keep consistency with the Italian Apple style.

### CODE REVIEW

File: Resources/it.lproj/ShortcutRecorder.strings [hunk 1]
```diff
@@ -35,7 +35,7 @@
 "The \"%@\" shortcut is already in use." = "La combinazione di tasti “%@“ è già in uso.";
 
 /* Value for the NSLocalizedDescriptionKey key of validator's system shortcuts error */
-"The \"%@\" shortcut can't be used because it's already used by a system-wide keyboard shortcut. If you really want to use this shortcut, most shortcuts can be changed in the Keyboard panel in System Preferences." = "La combinazione di tasti “%@“ non può essere usata perché è già usata come scorciatoia di sistema. Se vuoi veramente usare questa combinazione, molte scorciatoie possono essere cambiate dal pannello Tastiera o Mouse nelle Preferenze di Sistema.";
+"The \"%@\" shortcut can't be used because it's already used by a system-wide keyboard shortcut. If you really want to use this shortcut, most shortcuts can be changed in the Keyboard panel in System Preferences." = "La combinazione di tasti “%@“ non può essere usata perché già usata come abbreviazione di sistema. Se vuoi davvero usare questa combinazione di tasti, puoi modificare la maggior parte delle abbreviazioni nei pannelli Tastiera e Mouse delle Preferenze di Sistema.";
```
[the remark this answers is not in the document]
Comment (Kentzo) [reply Resources/it.lproj/ShortcutRecorder.strings:38]: Screen Shot 2020-05-26 at 11 01 56 PM
That's the panel that is being referred.

File: Resources/it.lproj/ShortcutRecorder.strings [hunk 2]
```diff
@@ -47,7 +47,7 @@
 "Option" = "Opzione";
 
 /* VoiceOver name of the Shift modifier key */
-"Shift" = "Maiuscole";
+"Shift" = "Maiuscolo";
```
[the remark this answers is not in the document]
Comment (ildede) [reply Resources/it.lproj/ShortcutRecorder.strings:50]: With this context in mind maybe is better to use the full italian text.
That is "Tasto maiuscolo", as mention on wikipedia [here](https://it.wikipedia.org/wiki/Tasto_maiuscolo).

But I'm not sure how Apple use this, do you know ho to activate the VoiceOver directly on MacOS and listen a similar part?

File: Resources/it.lproj/ShortcutRecorder.strings (hunk 1 above)
[the remark this answers is not in the document]
Comment (ildede) [reply Resources/it.lproj/ShortcutRecorder.strings:38]: I will fix this, thanks for the review!

File: Resources/it.lproj/ShortcutRecorder.strings (hunk 2 above)
[the remark this answers is not in the document]
Comment (Kentzo) [reply Resources/it.lproj/ShortcutRecorder.strings:50]: I think if you enable Voice Over and then navigate to System Preferences -> Keyboard -> Shortcuts you can find appropriate strings. Voice Over can be enabled in System Preferences -> Accessibility -> VoiceOver.

[the remark this answers is not in the document]
Comment (ildede) [reply Resources/it.lproj/ShortcutRecorder.strings:50]: I think that this is the most annoying thing that I ever heard... Anyway...

Actually "Maiuscole" is the word pronounced by VoiceOver, so for consistency with the OS is better to keep it.

[the remark this answers is not in the document]
Comment (ildede) [reply Resources/it.lproj/ShortcutRecorder.strings]: Solved by commit 6546d17

File: Resources/it.lproj/ShortcutRecorder.strings (hunk 1 above)
[the remark this answers is not in the document]
Comment (ildede) [reply Resources/it.lproj/ShortcutRecorder.strings]: Solved by commit 9908942

**Comment by Kentzo** [comment]
Thank you!

**Pull request merged**